cloudbuild/nightly/scripts/python-scripts/requirements.txt

```diff
@@ -19,7 +19,7 @@ grpc-google-iam-v1==0.12.7
 grpcio==1.59.3
 grpcio-status==1.59.3
 httplib2==0.22.0
-idna==3.6
```

Comment:
To fix PR [#106](https://github.com/GoogleCloudDataproc/flink-bigquery-connector/pull/106)